package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "scripts": {
     "test": "yarn lerna run test",
-    "release": "yarn lerna version minor --conventional-commits"
+    "release": "git push && yarn lerna version minor --conventional-commits"
```

Comment:
This seems unexpected.

Reply:
`lerna version` won't work unless your branch is pushed